api/rust/core/src/http/method.rs

```diff
@@ -8,10 +8,10 @@ pub enum Method {
 impl From<Method> for i32 {
 	fn from(field_type: Method) -> Self {
```

Comment:
```suggestion
	use Method::*;
	fn from(field_type: Method) -> Self {
```
Then you can remove the prefix from each match.  My main complaint with most Rust patterns.

Reply:
I'll take this suggestion, but we decided to change the API to be zero-indexed! You'll see this change in that PR.